### PR TITLE
Update teams-stream-troubleshooting.md

### PR DESCRIPTION
**Pull request by CarlosFarrica**
Added Diagnostic to validate if User can schedule Live Events. Similar to : https://learn.microsoft.com/en-us/microsoftteams/cloud-recording#meeting-recording-diagnostic-tools All Diags: https://learn.microsoft.com/en-us/microsoftteams/troubleshoot/teams-administration/admin-self-help-diagnostics

### CODE REVIEW

File: Teams/teams-stream-troubleshooting.md [hunk 1]
```diff
@@ -65,7 +65,11 @@ During live events many people will be joining to watch your event live. This co
 There are permissions across Microsoft Teams and Yammer that a user needs to be able to create a live event depending on which service you are using for the live event.
 
 1. Check that the Teams admin has enabled you to create live events.
-1. Check with your administrator that you have a valid Teams license that allows the creation of live events.
+2. Check with your administrator that you have a valid Teams license that allows the creation of live events.
+3. Select **Run Tests** below, which will populate the diagnostic in the Microsoft 365 Admin Center to help you validate if a user is able to schedule Teams live events.
```
Comment (JohanFreelancer9) [inline Teams/teams-stream-troubleshooting.md:69]:
```suggestion
3. Select the following **Run Tests**, which will populate the diagnostic in Microsoft 365 admin center to help you validate if a user is able to schedule Teams live events.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CarlosFarrica, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MicrosoftHeidi 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by lesahill** [comment]
@scanum @MicrosoftHeidi  - can you please merge this?
FYI @revaldiv

**Comment by scanum** [comment]
@MicrosoftHeidi
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**